packages/babel-preset-env/src/normalize-options.js

```diff
@@ -15,38 +15,52 @@ const validIncludesAndExcludes = new Set([
   ...defaultWebIncludes,
 ]);
 
-export const validateIncludesAndExcludes = (
-  opts: Array<string> = [],
+const pluginToRegExp = (plugin: any): RegExp => {
+  try {
+    return plugin instanceof RegExp
```

Comment:
Nitpick: Is there a chance of `plugin instanceof RegExp` throwing? If not, moving this out of the try-catch block may be a bit more readable. Not so important though.

Reply:
This `try/catch` is actually for `new RegExp`.

Reply:
I was thinking of something like this, but it doesn't really matter anyway. Don't worry about it.

```js
if (plugin instanceof RegExp) {
  return plugin;
}

try {
  return new RegExp(`^${normalizePluginName(plugin)}$`);
} catch (e) {
  return null;
}
```

Reply:
That's actually a good idea. Will change this and the doc and ask for a review again. Thank you. 👍